### ISSUE TITLE
Cannot install servicefabric on Linux, wrong packages dependencies

### ISSUE THREAD

**Issue by Jose-CF** [Azure/service-fabric-issues#220]
servicefabric package depends on dotnet-dev-1.0.0-preview2-003131.
However, that package is outdated and the dotnet-dev (non preview) should be installed instead.
But becaus servicefabrick package depends on the pewview one, it cannot install.

Expected: 
Dependency on the released package of dotnet-dev non-preview instead of the preview one.

**Comment by xunlu** [comment]
You are installing 5.5.0.2, right?  Are you following the instructions below, especially step 3?

https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-get-started-linux

**Comment by Jose-CF** [comment]
Yes, version 5.5.0.2. 
Yes, following the instructions.
Are you familiar with deb packages? The problem seems really obvious. servicefabric package depends on dotnet-dev-1.0.0-preview2-003131.
However dotnet-dev-1.0.0-preview2-003131 was already replaced with dotnet-dev-1.0.1.
If donet-dev-1.0.1 is installed (what is very recommended) then you cannot install dotnet-dev-1.0.0-preview2-003131 (and you should not... because it's an outdated version).

And therefore it is impossible to install servicefabric 5.5.0.2 because it depends on dotnet-dev-1.0.0-preview2-003131.

The fix is to update the package servicefabric 5.5.0.2 to depend on dotnet-dev-1.0.1 instead of dotnet-dev-1.0.0-preview2-003131.

Let me know.

**Comment by xunlu** [comment]
Our 5.5.0.2 version does depend on dotnet-dev-1.0.0-preview 2-003131 and has not moved to dotnet-dev-1.0.1 yet.  So to install 5.5.0.2 you need the preview version.  Of course, in the future we will move to newer version of dotnet, but not in 5.5.0.2.

**Comment by Jose-CF** [comment]
Thank you for your answer.

Do you have an schedule when you plan to drop the preview version? It is really sad to have to downgrade ourselves to a preview version of a software from Microsoft to use a software that comes also from Microsoft, i.e. isn't it a bit contradictory?

Is there like breaking changes that make it difficult to drop the preview version or is it matter of maintenance work?

In other words, if I force to install servicefabric 5.5.0.2 having dotnet-dev-1.0.1 sounds it like being impossible that it works?

**Comment by xunlu** [comment]
I'd not recommend that since all our testing has been on the preview version so far.  I will check with our dev working on dotnet integration to give you an estimate of the dates.   Thanks for the understanding.

**Comment by xunlu** [comment]
We are targeting the next release (middle May) for moving out of the preview version.

Thanks

**Comment by Jose-CF** [comment]
Thank you a lot for the information. 
And good luck with the moving!

**Comment by raunakpandya** [comment]
The latest C# SDK v0.9.0.0 can be used to develop apps using the non preview version.
